static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js

```diff
@@ -37,6 +37,16 @@ export type CtaUrls = {
     supportUrl: string,
 };
 
+export type ReaderRevenueRegion = 'gb' | 'us' | 'au' | 'row';
+
+const getReaderRevenueRegion = (geolocation: string): ReaderRevenueRegion => {
+    const location = geolocation.toLowerCase();
```

Comment:
is there any need to make them lowercase when they're always uppercase in dotcom?

Reply:
as discussed in person - maybe full words are better to distinguish from genuine country codes? `'united-kingdom' | 'united-states' | 'australia' | 'rest-of-world' `

Reply:
This was to make them distinct from the geolcation (although they are determined by geolocation) to reduce chance they were accidentally used. I think your suggested making them clearly different by using words is a good one, especially as rest-of-world would not longer be confusing row 🚣